### PR TITLE
Basic Authentication Guide

### PR DESCRIPTION
**Pull request by faonbr**
@samcrichard , here is the basic Authentication guide you asked me.

### CODE REVIEW

**Review by samcrichard — DISMISSED** [review]
legacy is misspelled, support is misspelled, code changes should be plural

File: docs/guides/authn-with-ngrok.md [hunk 1]
```diff
@@ -86,12 +86,41 @@ Use the `--oidc`, `--oidc-client-id` and `--oidc-client-secret` flags with your
...
+    addr: 3000
+    issuer_url: https://accounts.google.com/o/oauth2/v2/auth
+    client_id: 806764744727-3j3t4n6s1m1hna6ahlq08q555a4j06t5.apps.googleusercontent.com
+    client_secret: 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
+    scopes: *
+```
+
+1. Run the ngrok agent using the following command:
+```bash
+ngrok start google
```
Comment (lee-tschetter-ngrok) [inline docs/guides/authn-with-ngrok.md]: correct command is:

ngrok start oidc

**Comment by SafeArrow816** [comment]
SafeArrow 816

**Comment by SafeArrow816** [comment]
SafeArrow 816 change.

**Comment by SafeArrow816** [comment]
Merging

File: docs/guides/authn-with-ngrok.md (hunk 1 above)
Comment (SafeArrow816) [reply docs/guides/authn-with-ngrok.md]: Pull request.

**Comment by samcrichard** [comment]
Basic looks awesome. Can you add it to the boxes under secure?

**Comment by faonbr** [comment]
> Basic looks awesome. Can you add it to the boxes under secure?

Done.

**Pull request closed without merging**